src/Microsoft.Health.CustomerManagedKey/Client/EmptyKeyClient.cs

```diff
@@ -0,0 +1,13 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Azure.Security.KeyVault.Keys;
+
+namespace Microsoft.Health.CustomerManagedKey.Client;
+
+internal class EmptyKeyClient : KeyClient
```

Comment:
This actually wasn't included for test purposes - In `CustomerKeyClientFactory`, if I return `null` when CMK is not enabled, DI will complain because I'm trying to inject a null value for KeyClient. So I made this to be used for when CMK is not enabled. Not sure if theres a better way to do this instead

Reply:
Talked offline; ended up creating the KeyClient in the test itself, so we don't need to worry about creating a null KeyClient